tox.ini

```diff
@@ -1,18 +1,19 @@
 [tox]
-envlist = py3-docs
+envlist = docs
 skipsdist = true
+requires =
+  tox-pip-sync
+  tox-pyenv
+  tox-run-command
 
 [testenv]
 skip_install = true
 deps =
-    {docs,checkdocs}: recommonmark
-    {docs,checkdocs}: sphinx-autobuild
-    {docs,checkdocs}: sphinx
-    {docs,checkdocs}: sphinx_rtd_theme
+    {docs,checkdocs}: -r requirements/docs.txt
```

Comment:
We now pin all our Python requirements for a few reasons:

 * So you can't get weirdness where it passes in CI but not locally or vice versa
 * So dependabot can poke us